### PR TITLE
Update pstn-usage-report.md

### PR DESCRIPTION
**Pull request by msmikedavis**
Export to excel changed to COMMA delimited file, so removed TAB delimiter statements.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: Skype/SfbOnline/skype-for-business-online-reporting/pstn-usage-report.md [hunk 1]
```diff
@@ -99,10 +99,8 @@ This is what the report looks like.
...
  ***
-![Number 3](../images/sfbcallout3.png)<br/>You can also export the report data into a TAB delimited Excel file, by clicking or tapping the **Export to Excel** button. You can export data up to one year from the current date.<br/><br/> This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 users, you can sort and filter within the table in the report itself. 
-    > [!Note] 
-    > Despite the export file named as .CSV (which implies a comma delimited export), as there may be commas in the data set, the file is actually delimited with **TABS** and not **COMMAS**.
-
+![Number 3](../images/sfbcallout3.png)<br/>You can also export the report data into a COMMA delimited Excel file, by clicking or tapping the **Export to Excel** button. You can export data up to one year from the current date unless country specific regulation prohibit retention of the data for 12 months.<br/><br/> This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 users, you can sort and filter within the table in the report itself.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/skype-for-business-online-reporting/pstn-usage-report.md]:
```suggestion
![Number 3](../images/sfbcallout3.png)<br/>You can also export the report data into a COMMA delimited Excel file, by clicking or tapping the **Export to Excel** button. You can export data up to one year from the current date unless country specific regulation prohibits retention of the data for 12 months.<br/><br/> This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 users, you can sort and filter within the table in the report itself. 
```

**Comment by scanum** [comment]
@tonysmit 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**